src/vnc.yml

```diff
@@ -1,37 +1,101 @@
 ---
 - hosts: all
-  name: Install VNC
+  name: Install VNC and configure vnc user
```

Comment:
`vnc user` and `VNC user` are both being used. I would like us to settle on one and be consistent.

Reply:
See commit 44326ad.